gnome-initial-setup/eos-setup-demo-user-autologin

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# eos-setup-demo-user-autologin - set the demo user up to auto-login
+# Copyright (C) 2016-2017 Endless Mobile, Ltd.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import argparse
+import sys
+
+from gi.repository import GLib
+
+GDM_DAEMON_CONF = '/etc/gdm3/daemon.conf'
+
+
+def main(arguments):
+    keyfile = GLib.KeyFile.new()
+    keyfile.load_from_file(GDM_DAEMON_CONF, GLib.KeyFileFlags.NONE)
+    keyfile.set_string('daemon', 'TimedLoginEnable', 'true')
+    keyfile.set_string('daemon', 'TimedLogin', arguments.user)
+    keyfile.set_string('daemon', 'TimedLoginDelay', '1')
+    keyfile.save_to_file(GDM_DAEMON_CONF)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser('Set up timed login for demo user')
```

Comment:
What's the difference between timed login and automatic login? Since we set the user to automatically login when we create the user, do we also need this one?